### PR TITLE
APPS-3467: master symfony container

### PR DESCRIPTION
**Pull request by vol4onok**
https://spryker.atlassian.net/browse/APPS-3467

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env php
+<?php
+
+use SprykerSdk\Kernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+if (!is_file(dirname(__DIR__).'/vendor/autoload_runtime.php')) {
+    throw new LogicException('Symfony Runtime is missing. Try running "composer require symfony/runtime".');
```
Comment (dereuromark) [inline bin/console:8]:
```suggestion
    throw new LogicException('Symfony Runtime is missing. Try running `composer require symfony/runtime`.');
```
Code pieces should always be code fenced, not quoted.

File: codeception.yml [hunk 2]
```diff
@@ -25,34 +29,48 @@ suites:
         class_name: SprykTester
         modules:
             enabled:
-                - SprykerSdkTest\Module\DevelopmentModeModule:
-                    developmentMode: 'core'
+                - Symfony:
+                      part: services
+                      app_path: 'src'
```
Comment (dereuromark) [inline codeception.yml:34]: Paths should ideally always have trailing slash /

File: composer.json [hunk 3]
```diff
@@ -4,25 +4,33 @@
...
+        "symfony/framework-bundle": "^5.4.0 || ^6.0.0",
+        "symfony/proxy-manager-bridge": "^5.4.0",
+        "symfony/runtime": "^5.3.0 || ^6.0.0",
+        "symfony/yaml": "^5.0.0 || ^6.0.0",
+        "symplify/autowire-array-parameter": "^9.3.0",
+        "twig/twig": "^3.0.0"
     },
     "require-dev": {
         "codeception/codeception": "^4.0.0",
-        "phpstan/phpstan": "^0.12.98",
```
Comment (dereuromark) [inline composer.json:24]: I would rather keep phpstan than rector actually.

File: codeception.yml (hunk 2 above)
Comment (vol4onok) [reply codeception.yml:34]: with slash ./src /src it always goes to ../src 
image

**Pull request closed without merging**